spec/abstractops.html

```diff
@@ -950,12 +955,12 @@ <h1>ISO 8601 grammar</h1>
           TimeZoneIANANameComponent `/` TimeZoneIANANameTail
 
       TimeZoneIANAName :
+          `Etc/GMT` ASCIISign UnpaddedHour
+          TimeZoneUTCOffsetName
           TimeZoneIANANameTail
 
       TimeZoneBracketedName :
           TimeZoneIANAName
-          `Etc/GMT` ASCIISign Hour
```

Comment:
I think we can remove TimeZoneBracketedName altogether now?

This change also affects step 4 of [ParseTemporalTimeZoneString](https://tc39.es/proposal-temporal/#_ref_2536) but I believe that is also a correct change.

Reply:
right, we can remove `TimeZoneBracketedName` -- done

I'm not sure I understand the comment about `ParseTemporalTimeZoneString`. Can you elaborate?

Reply:
Sure! Here's ParseTemporalTimeZoneString step 4:

> Let each of _z_, _offsetString_, and _name_ be the source text matched by the respective _UTCDesignator_, _TimeZoneNumericUTCOffset_, and _TimeZoneIANAName_ Parse Node enclosed by _parseResult_, or an empty sequence of code points if not present.

This change affects that step because the change in _TimeZoneIANAName_ means that now _name_ can end up being one of the `Etc/GMT±n` strings as well as a `±hh:mm` offset string, which it could not before. But I believe that was a mistake, which this change now fixes.

Reply:
I've cleaned things up by reverting the proposed grammar cleanup. (`validStrings.mjs` was cleaned up, too.)